src/CarrierLogo/index.js

```diff
@@ -0,0 +1,129 @@
+// @flow
+import * as React from "react";
+import * as tokens from "orbit-design-token";
+
+const baseURL = "//images.kiwi.com"; // token or const
+
+const directorySizes = {
+  small: 16,
+  medium: 32,
+  large: 32,
+};
+
+const renderSizes = {
+  small: parseInt(tokens.heightIconSmall, 10),
+  medium: parseInt(tokens.heightIconMedium, 10),
+  large: 32, // replace it for token heightIconLarge
+};
+
+const retinaSizes = {
+  small: 32,
+  medium: 64,
+  large: 64,
+};
+
+type Props = {
+  size: "small" | "medium" | "large",
+  carriers: { code: string, name: string }[],
+};
+
+const CarrierLogo = (props: Props) => {
+  const { size, carriers } = props;
+
+  let sourceSize = directorySizes[size];
+  let imageSize = renderSizes[size];
+  let srcSetSize = retinaSizes[size];
+
+  if (carriers.length > 1) {
+    sourceSize = directorySizes.small;
+    imageSize = renderSizes.small;
+    srcSetSize = retinaSizes.small;
+  }
+
+  return (
+    <div>
+      {carriers
+        .slice(0, 4)
+        .map(carrierImage => (
+          <img
+            key={carrierImage.code}
+            src={`${baseURL}/airlines/${sourceSize}/${carrierImage.code}.png`}
```

Comment:
I would suggest enabling accepting the whole URL as well => `{ code: string, name: string }` => `{ code: string, name: string, url?: string }` 

and then something like ```url || `${baseURL}/airlines/${sourceSize}/${carrierImage.code}.png` ```

Reply:
ok, I see your explanation